### PR TITLE
Removed google.apputils usage in stubs

### LINKED ISSUE #1576: forseti_enforcer is not working
```
henry@forseti-server-vm-4330:/home/ubuntu/forseti-security/google/cloud/forseti/enforcer$ forseti_enforcer --enforc
e_project inventoryscanner-henry --policy_file /home/ubuntu/forseti-security/samples/enforcer/default_allow_policy.json 
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/forseti_security-2.0.0-py2.7.egg/google/cloud/forseti/enforcer/enforcer.py", line 27, in <module>
    from google.cloud.forseti.common.util import file_loader
ImportError: No module named cloud.forseti.common.util

henry@forseti-server-vm-4330:/home/ubuntu/forseti-security/google/cloud/forseti/enforcer$ forseti_enforcer
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/forseti_security-2.0.0-py2.7.egg/google/cloud/forseti/enforcer/enforcer.py", line 27, in <module>
    from google.cloud.forseti.common.util import file_loader
ImportError: No module named cloud.forseti.common.util
```

### PR DESCRIPTION
**Pull request by joecheuk**
Resolves #1576 

We are have conflicting namespace that's why we are getting import errors in some of the python classes.

### CODE REVIEW

File: google/cloud/forseti/services/cli.py [hunk 1]
```diff
@@ -1193,5 +1196,4 @@ def get_config_path():
 
 
 if __name__ == '__main__':
-    ENV_CONFIG = DefaultConfigParser.load()
-    main(sys.argv[1:], ENV_CONFIG)
```
Comment (blueandgold) [inline google/cloud/forseti/services/cli.py]: qq: why is this change necessary?

Comment (joecheuk) [reply google/cloud/forseti/services/cli.py]: I just want to keep it consistent as the server and the enforcer both have their main() methods self contained.
